Sources/HAP/Server/Device.swift

```diff
@@ -77,14 +81,14 @@ public class Device {
     ///   - accessories: accessories to be bridged
     convenience public init(
         bridgeInfo: Service.Info,
-        setupCode: String,
         storage: Storage,
-        accessories: [Accessory]) {
+        accessories: [Accessory],
+        setupCode: SetupCode = .automatic) {
```

Comment:
Can we move the argument to where it was? That means the API doesn't break as much.

Reply:
OK